pkg/vpcmodel/explainabilityInput.go

```diff
@@ -47,7 +47,7 @@ const noValidInputMsg = "does not represent a legal IP address, a legal CIDR or
 // getVPCConfigAndSrcDstNodes given src, dst names returns the config in which the exaplainability analysis of these
 // should be done and the Nodes for src and dst.
 
-// src and dst when referring to a vsi *name* may be prefixes with their vpc name to solve ambiguity
+// src and dst when referring to a vsi *name* may be prefixes "/" with their vpc name to solve ambiguity
```

Comment:
documentation should also be added to the cli args, for valid input format of vsi of the explainability option

Reply:
right. done.